quick_tour/the_big_picture.rst

```diff
@@ -131,18 +131,18 @@ of database calls, HTML tags and other PHP code in the same script. To achieve
 this goal with Symfony, you'll first need to learn a few fundamental concepts.
 
 When developing a Symfony application, your responsibility as a developer
-is to write the code that maps the user's *request* (e.g.  ``http://localhost:8000/app/example``)
+is to write the code that maps the user's *request* (e.g.  ``http://localhost:8000/``)
```

Comment:
while it wasn't your fault, can you please remove one of the 2 spaces before the URL?